lib/lotus/model.rb

```diff
@@ -27,6 +27,16 @@ class NonPersistedEntityError < ::StandardError
     class InvalidMappingError < ::StandardError
     end
 
+    # Error for invalid query
+    # It's raised when a query is malformed
+    #
+    # @since 0.3.1
+    class InvalidQueryError < ::StandardError
+      def initialize(message = nil)
```

Comment:
Actually, there is :cry: When thrown from the memory adapter, the message would be cryptic, as @jodosha pointed out. In this case, I thought the default message would be better than `InvalidQueryError: undefined method `>' for nil:NilClass`